.github/workflows/automated-acceptance-tests.yml

```diff
@@ -20,43 +21,113 @@ env:
         uses: aws-actions/configure-aws-credentials@v4
         with:
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
-          role-to-assume: ${{ secrets.AWS_RELEASE_ROLE }}
+          role-to-assume: ${{ secrets.AWS_AAC_NS_ROLE }}
 
       - name: Build Superwerker
         run: |
-          aws sts get-caller-identity
           yarn global add aws-cdk
+          yarn install
           yarn test
           yarn synth
           yarn publish-assets
         env:
           SUPERWERKER_VERSION: 0.0.0-AAC
+          NODE_ENV: development
         shell: bash
         working-directory: cdk
 
-      - name: Configure AWS credentials
+      - name: Assume Pipeline AWS Role
         uses: aws-actions/configure-aws-credentials@v4
         with:
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
           role-to-assume: ${{ secrets.AWS_AAC_ROLE }}
 
       - name: Deploy Superwerker
+        id: deploy-superwerker
         run: |
-          aws sts get-caller-identity
-          aws cloudformation deploy --stack-name superwerker --template-file ./cdk.out/SuperwerkerStack.template.json --parameter ParameterKey=Domain,ParameterValue=${{ secrets.SUPERWERKER_DOMAIN }} ParameterKey=Subdomain,ParameterValue=${{ inputs.account }} ParameterKey=NotificationsMail,ParameterValue=root+notifications@${{ inputs.account }}.${{ secrets.SUPERWERKER_DOMAIN }} --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM CAPABILITY_NAMED_IAM --disable-rollback  --no-cli-pager
+          nohup aws cloudformation deploy --stack-name superwerker --template-file ./cdk.out/SuperwerkerStack.template.json --parameter-overrides Domain=${SUPERWERKER_DOMAIN} Subdomain=${{ inputs.account }} NotificationsMail=root+notifications@${{ inputs.account }}.${SUPERWERKER_DOMAIN} --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM CAPABILITY_NAMED_IAM --disable-rollback  --no-cli-pager &
+          while ! DOMAIN_NAME_SERVERS=$(aws ssm get-parameter --name /superwerker/domain_name_servers --query Parameter.Value --output text 2>/dev/null); do sleep 10; done
+          echo "DOMAIN_NAME_SERVERS=${DOMAIN_NAME_SERVERS}" >> "$GITHUB_OUTPUT"
         env:
           SUPERWERKER_VERSION: 0.0.0-AAC
+          SUPERWERKER_DOMAIN: ${{ secrets.SUPERWERKER_DOMAIN }}
         shell: bash
         working-directory: cdk
+
+      - name: Assume Superwerker AWS Role
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
+          role-to-assume: ${{ secrets.AWS_AAC_NS_ROLE }}
+      
+
+      - name: Configure Subdomain Delegation
+        run: |
+          echo "Configuring subdomain delegation to nameservers ${DOMAIN_NAME_SERVERS}..." 
+          aws cloudformation deploy --stack-name superwerker-pipeline-dns-wiring-pipeline02 --template-file ../tests/pipeline-dns-wiring.yaml --parameter-overrides RootMailDelegationTarget=${DOMAIN_NAME_SERVERS} RootMailDomain=${SUPERWERKER_DOMAIN} RootMailSubdomain=${{ inputs.account }}
```

Comment:
is there a reason why the stack name is hardcoded to -pipeline02?

Reply:
No. Good catch.